src/dali_executor/CMakeLists.txt

```diff
@@ -28,11 +28,30 @@ set(
 
 include(${tritondalibackend_SOURCE_DIR}/cmake/dali.cmake)
 
+string(CONCAT DALI_DOWNLOAD_CUDA_VERSION ${CUDAToolkit_VERSION_MAJOR} ${CUDAToolkit_VERSION_MINOR})
+if (${CUDAToolkit_VERSION_MAJOR} GREATER_EQUAL 11)
```

Comment:
I believe triton doesn't support earlier CUDA version, so we don't really need this. Latest cuda is fine.